src/Features/ShippingLabelBanner.php

```diff
@@ -126,17 +126,18 @@ private function count_shippable_items( \WC_Order $order ) {
 	 * @param string $hook current page hook.
 	 */
 	public function add_print_shipping_label_script( $hook ) {
+		$rtl = is_rtl() ? '.rtl' : '';
 		wp_enqueue_style(
 			'print-shipping-label-banner-style',
-			Loader::get_url( 'print-shipping-label-banner/style.css' ),
-			array(),
+			Loader::get_url( "print-shipping-label-banner/style{$rtl}.css" ),
+			array( 'wp-components' ),
```

Comment:
`client/wp-admin-scripts/print-shipping-label-banner/setup-notice/index.js` still uses `wp-components`, shouldn't this be inside `wp_enqueue_script`?

Reply:
`wp-components` is actually a stylesheet dependency so it needs to be in the styles deps. It's exactly what the [doc says](https://developer.wordpress.org/block-editor/packages/packages-components/#usage).